macos/snd_.ksy

```diff
@@ -0,0 +1,201 @@
+meta:
+  id: snd_
+  endian: be
```

Comment:
All specs in kaitai_struct_formats [need to include a `license` in the metadata](https://github.com/kaitai-io/kaitai_struct_formats/blob/master/CONTRIBUTING.md#contributing).

Reply:
added MIT license